java/piranha/src/main/java/com/uber/piranha/config/Config.java

```diff
@@ -124,16 +126,33 @@ private Config(
   }
 
   /**
-   * Return all configuration method records matching a given method name.
+   * Return all configuration method records matching a given method name. A method record name
+   * field can be used to express a single method invocation or a chain of two method invocations. A
```

Comment:
is the chain restricted to two?

Reply:
Yea. We can go `n` too, but it will have performance implications. Maybe we can accept method chain length as an input (default = 1) instead of adding a flag for "chaining of methods" (As u mentioned in your below comment) ?

Reply:
I decided to keep it to `2` for now, until we need more.